scripts/meson.build

```
@@ -1,16 +1,20 @@
-prog_python = import('python').find_installation('python3', required : get_option('collector'), modules : ['cffi'])
+prog_python = import('python').find_installation(
+  'python3',
+  required : get_option('collector'),
+  modules : ['cffi'],
+  disabler : true,
+)
+collector_script = files(meson.current_source_dir() / 'collect_stats.py')
```

```suggestion
collector_script = files('collect_stats.py')
```
Using `meson.current_source_dir()` in `files()` is not required.

---

Thanks!